### DISCUSSION TITLE
3D case

### DISCUSSION THREAD

**Discussion by ikoz** [klaasnicolaas/home-assistant-glow#127] [General]
I discovered, unfortunatelly after printing, that the STL of case included in the repository is for ESP32 specifically. However I am using an ESP8266 (NodeMCU) for this, so it doesn't really fit in the case, as it is slightly smaller, and the supports are in different locations. Would be great to have ESP8266 case designs and more clear naming on the files.

**Comment by klaasnicolaas** [comment]
Good point, I could certainly improve on that
